### DISCUSSION TITLE
Wallet Status:  Not Synced (792,992) - but full node is synced

### DISCUSSION THREAD

**Discussion by majidhirad** [Chia-Network/chia-blockchain#8482] [Support]
hi
delete  wallet db but still not syncing in 7 days
i cant withdraw my chia

**Comment by majidhirad** [comment]
PS C:\Users\Hirad> chia show -s
Current Blockchain Status: Full Node Synced

Peak: Hash: d1231bb9c7a893f6713fd36bf27f3e6b221af3ed841998b31d3aa22be01eba30
      Time: Tue Sep 14 2021 20:42:39 Romance Summer Time                  Height:     859057

Estimated network space: 33.814 EiB
Current difficulty: 2928
Current VDF sub_slot_iters: 136314880
Total iterations since the start of the blockchain: 3094182896576

  Height: |   Hash:
   859057 | d1231bb9c7a893f6713fd36bf27f3e6b221af3ed841998b31d3aa22be01eba30
   859056 | 41abf45fcd9d01a18407050d8aa4712d396512124548cf66828ebf3b528c99dc
   859055 | ed784201581152e2df4332febb40da022a705b05d367f06e7335337e1b321abb
   859054 | 45fc9bcdaa4cac2784e248f7c73e93219f7204d08854826b69aaf21c248802f4
   859053 | fb43857df2e505df68dda8b16b5fd395b6c0ee5c719f3fc699cf3a9fb47cb32d
   859052 | ba7b79675e700cbd4858762f71a496e8dea1147b0680b87af6aa90158fa0808d
   859051 | 37b9a7e58f6048ad8d0c78542faff1ff2b733a76f2613ebdb237b3afd47bacf1
   859050 | 571e9ebd6b2d8faa31491293b1e065ebab2b83013b3a5ffe8da97a49e6be8979
   859049 | ee5965b2952727a826a1bcd72b9d2ab7eaa46ee381b633ccb7cf7918bc52d8e4
   859048 | a7b383b5518f5ef39d92ed969337ec7341817d6528fe53520ea17589c252c151


PS C:\Users\Hirad> chia wallet show
Wallet height: 792992
Sync status: Not synced
Balances, fingerprint: 1693159549
Wallet ID 1 type STANDARD_WALLET Chia Wallet
   -Total Balance: 4.000025205431 xch (4000025205431 mojo)
   -Pending Total Balance: 4.000025205431 xch (4000025205431 mojo)
   -Spendable: 4.000025205431 xch (4000025205431 mojo)
Wallet ID 2 type POOLING_WALLET Pool wallet
   -Total Balance: 0.0 xch (0 mojo)
   -Pending Total Balance: 0.0 xch (0 mojo)
   -Spendable: 0.0 xch (0 mojo)

**Comment by jack60612** [comment]
The wallet syncs separately from the node

**Comment by loppefaaret** [comment]
im going to transfer this into a support discussion, and see if we can figure out what exactly is going on with your wallet here

**Reply by loppefaaret** [reply]
if you open up a file explorer window, and browse to `C:\Users\<USERNAME>\.chia\mainnet\log\` you will find a file called `debug.log` - can you see any ERROR reported by the wallet in here ?

**Reply by ItsMrKnowItAll** [reply]
I have the same issue. I'm updating my post with logs.

**Comment by BrandtH22** [comment]
@majidhirad , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

Note - newer version of the chia wallet use the light wallet protocol which generally leads to the wallet syncing rather quickly (especially when there is a local trusted full node), you will still find that the wallet has to sync from full nodes though so it can be syncing while the full node indicates synced.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
